src/main/kotlin/org/cmapi/primitives/GeoBase.kt

```diff
@@ -5,9 +5,9 @@ import multiplatform.expected.MpUUID
 
 open class GeoBase : IGeoBase
 {
-    override var name = ""
-    override var geoId: MpUUID = GetUUIDFunction.randomUUID()
-    override var dataProviderId: String = ""
-    override var description = ""
-    override val properties : HashMap<String, String> = hashMapOf()
+    override var name : String? = ""
+    override var geoId: MpUUID? = GetUUIDFunction.randomUUID()
+    override var dataProviderId: String? = null
+    override var description : String? = ""
```

Comment:
We have a `String?` being initialized with null and two being initialized with `""`.  Which should be the default way of handling nullable Strings in the API?  Choose one and let's stick with it for this API.

My gut feeling is a `String` doesn't need to be nullable since empty is probably what test-basic wants to do anyway, but you know the specific constraints of the EMP3 tests right now.

Reply:
So in this specific class in the empty constructor they initialized name and description with "" and did not initialize dataProviderId (therefore it is null). If I change it to null, then it won't match the exactly as before.  This is why I missed these null values before.  I am not sure why they made these decisions of initializing some strings to null and others to ""

Reply:
So changing line 10 to `""` won't work with emp3-android?

Reply:
I created a ticket #4 and will address this on the next version